DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/tools/manual-capture/how-to-activate-manual-capture.md

---
## Activation

> The activation of Manual Capture takes place in the MultiSafepay Control. Email our Screening Team at <risk@multisafepay.com>

Reviewer: ```suggestion
To activate Manual Capture in your MultiSafepay Control, email the Risk Team at <risk@multisafepay.com>
```

After our screening department has enabled this feature for the website you selected, it is important to view the [API documentation](/api/#manual-capture-authorization) regarding Manual Capture. This is not incorporated in our [ecommerce integrations](/integrations/ecommerce-integrations) by default, but it can be added through customization from your side.

> Do you have any questions about this process? Email our Integration Team at <integration@multisafepay.com>

Reviewer: ```suggestion
For any questions, email the Integration Team at <integration@multisafepay.com>
```
